cognite/neat/_rules/analysis/_base.py

```diff
@@ -524,6 +549,63 @@ def class_uri(self, class_: ClassEntity) -> URIRef | None:
         else:
             return None
 
+    def property_uri(self, property_: InformationProperty) -> URIRef | None:
+        if (instance_source := property_.instance_source) and isinstance(instance_source.traversal, SingleProperty):
+            prefix = instance_source.traversal.property.prefix
+            suffix = instance_source.traversal.property.suffix
+
+            if namespace := self.information.prefixes.get(prefix):
+                return namespace[suffix]
+        return None
+
     @property
     def multi_value_properties(self) -> list[InformationProperty]:
         return [prop_ for prop_ in self.information.properties if isinstance(prop_.value_type, MultiValueTypeInfo)]
+
+    @property
+    def query_config_by_view_id(
+        self,
+    ) -> dict[dm.ViewId, dict[str, None | URIRef | dict[URIRef, str]]]:
```

Comment:
Maybe worth a dataclass? This nested dict is a bit hard to use as you have to know that is should have two keys 'property_renaming_config' and ''rdf_type'.